### PR TITLE
fuzzers: Fix error usage

### LINKED ISSUE #791: Fix error usage in 052-pip-clkin
The TCL `error` function must be called with an error message, IE
```tcl
error "Failure to work"
```
The following TCL code is invalid;
```tcl
if {$tries >= 3} {
    puts "WARNING: failed to route net after $tries tries"
    error
}
```

You can see it in the error output for 052-pip-clkin output;
```
052-pip-clkin- 30m53s: wrong # args: should be "error message ?errorInfo? ?errorCode?"
052-pip-clkin- 30m53s:     while executing
052-pip-clkin- 30m53s: "error"
052-pip-clkin- 30m53s:     invoked from within
052-pip-clkin- 30m53s: "if {$tries >= 3} {
052-pip-clkin- 30m53s:             puts "WARNING: failed to route net after $tries tries"
052-pip-clkin- 30m53s:             error
052-pip-clkin- 30m53s:         }"  
```

It needs to be something like;
```tcl
if {$tries >= 3} {
    puts "WARNING: failed to route net after $tries tries"
    error ""
}
```
or
```tcl
if {$tries >= 3} {
    error "WARNING: failed to route net after $tries tries"
}
```

### PR DESCRIPTION
**Pull request by tmichalak**
Signed-off-by: Tomasz Michalak <tmichalak@antmicro.com>

This PR fixes #791

### CODE REVIEW

File: fuzzers/052-pip-clkin/generate.tcl [hunk 1]
```diff
@@ -90,8 +90,7 @@ for {set idx 0} {$idx < [llength $todo_lines]} {incr idx} {
 
         # sometimes it gets stuck in specific src -> dst locations
         if {$tries >= 3} {
-            puts "WARNING: failed to route net after $tries tries"
-            error
+            error "WARNING: failed to route net after $tries tries"
```
Comment (mithro) [inline fuzzers/052-pip-clkin/generate.tcl]: This is probably better;

```suggestion
            error "ERROR: failed to route net after $tries tries"
```

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl [hunk 2]
```diff
@@ -72,7 +72,7 @@ proc find_src_pin {tile src_wire} {
             return $clb_src_pin
         }
     }
-    error
+    error ""
```
Comment (mithro) [inline fuzzers/051-pip-imuxlout-bypalts/generate.tcl]:
```suggestion
    error "ERROR: Unable to find source pin for $tile/$src_wire (iterations: $iterations)"
```

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl [hunk 3]
```diff
@@ -48,7 +48,7 @@ proc find_dst_pin {tile dst_wire} {
             return $clb_dst_pin
         }
     }
-    error
+    error ""
```
Comment (mithro) [inline fuzzers/051-pip-imuxlout-bypalts/generate.tcl]:
```suggestion
    error "ERROR: Unable to find destination pin for wire $tile/$dst_wire (iterations: $iterations)"
```

File: fuzzers/053-pip-ctrlin/generate.tcl [hunk 4]
```diff
@@ -86,8 +86,7 @@ for {set idx 0} {$idx < [llength $todo_lines]} {incr idx} {
 
         # sometimes it gets stuck in specific src -> dst locations
         if {$tries >= 3} {
-            puts "WARNING: failed to route net after $tries tries"
-            error
+            error "WARNING: failed to route net after $tries tries"
```
Comment (mithro) [inline fuzzers/053-pip-ctrlin/generate.tcl]:
```suggestion
            error "ERROR: failed to route net after $tries tries"
```

File: fuzzers/055-pip-gnd/generate.tcl [hunk 5]
```diff
@@ -87,8 +87,7 @@ for {set idx 0} {$idx < [llength $todo_lines]} {incr idx} {
 
         # sometimes it gets stuck in specific src -> dst locations
         if {$tries >= 3} {
-            puts "WARNING: failed to route net after $tries tries"
-            error
+            error "WARNING: failed to route net after $tries tries"
```
Comment (mithro) [inline fuzzers/055-pip-gnd/generate.tcl]:
```suggestion
            error "ERROR: failed to route net after $tries tries"
```

Comment (tmichalak) [reply fuzzers/055-pip-gnd/generate.tcl]: Done

File: fuzzers/053-pip-ctrlin/generate.tcl (hunk 4 above)
Comment (tmichalak) [reply fuzzers/053-pip-ctrlin/generate.tcl]: Done

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl (hunk 3 above)
Comment (tmichalak) [reply fuzzers/051-pip-imuxlout-bypalts/generate.tcl]: Done

File: fuzzers/051-pip-imuxlout-bypalts/generate.tcl (hunk 2 above)
Comment (tmichalak) [reply fuzzers/051-pip-imuxlout-bypalts/generate.tcl]: Done

File: fuzzers/052-pip-clkin/generate.tcl (hunk 1 above)
Comment (tmichalak) [reply fuzzers/052-pip-clkin/generate.tcl]: Done

**Pull request merged**